src/lxc/network.c

```diff
@@ -183,6 +224,20 @@ static int instantiate_veth(struct lxc_handler *handler, struct lxc_netdev *netd
 		goto out_delete;
 	}
 
+	/* setup ipv4 routes on the host interface */
+	if (lxc_setup_ipv4_routes(&netdev->ipv4_routes, netdev->priv.veth_attr.ifindex)) {
+		errno = -err;
+		ERROR("Failed to setup ipv4 routes for network device \"%s\"", veth1);
+		goto out_delete;
+	}
+
+	/* setup ipv6 routes on the host interface */
+	if (lxc_setup_ipv6_routes(&netdev->ipv6_routes, netdev->priv.veth_attr.ifindex)) {
+		errno = -err;
```

Comment:
Where is that err initialized?

Reply:
@brauner I actually dont need those as errno is set inside `lxc_setup_ipv6_routes`